README.rst

```diff
@@ -61,7 +61,11 @@ Usage
 #. (Optional) Sink course data from the LMS to clickhouse (see  `https://github.com/openedx/openedx-event-sink-clickhouse` for more information)::
 
     tutor [dev|local] do dump-courses-to-clickhouse --options "--force"
+    # If you already have some courses in your clickhouse sink, its better to drop --options "--force" as it will create duplicates of the pre-existing courses.
 
+#. (Optional) Sink Historical event data to ClickHouse::
+
```

Comment:
Thanks for this! Can you add a line explaining that this will work for a default Tutor install, but if you're not storing all of your tracking logs in one file on disk you'll need the full instructions here? https://event-routing-backends.readthedocs.io/en/latest/howto/how_to_bulk_transform.html

Reply:
Done